lib/controllers/cats.js

```diff
@@ -1,16 +1,19 @@
 const { Router } = require('express');
-const { cats } = require('../../data/cats');
-
+const Cat = require('../models/cat');
 
 module.exports = Router()
-  .get('/:id', (req, res) => {
+  .get('/:id', async(req, res) => {
     const id = req.params.id;
-    const matchingCat = cats.find((cat) => cat.id === id);
+    const matchingCat = await Cat.getById(id);
     res.json(matchingCat); 
   })  
-  .get('/', (req, res) => {
-    const cat = cats.map(({ id, name }) => {
-      return { id, name };
+  .get('/', async (req, res) => {
+    const dataCats = await Cat.getAll();
+    const finalData = dataCats.map(({ id, name }) => {
+      return{
+        id,
+        name
+      };
```

Comment:
since the `getAll()` returns an array we don't need to map over it again. If you really just want to return the id and name you can change your SQL query to be: `SELECT id, name FROM cats` and that will get you an array of the id's and names for that route. Saves you some munging in the controller.